commit 2cc8c240f6ea6ba7fae15721c9d7892024c65a8f
Author: Valentina Mukhamedzhanova <vmukhame@redhat.com>
Date:   Thu Aug 28 10:17:38 2014 +0200

    reposync: show urls for all repos when using -u option. BZ 1133125

diff --git a/reposync.py b/reposync.py
index 6500db6..bb4c59d 100755
--- a/reposync.py
+++ b/reposync.py
@@ -283,7 +283,7 @@ def main():
         if opts.urls:
             for pkg in download_list:
                 print urljoin(pkg.repo.urls[0], pkg.relativepath)
-            return 0
+            continue
 
         # create dest dir
         if not os.path.exists(local_repo_path):
